### PR TITLE
Completed prep-javascript-objects - Ahmad

### PR DESCRIPTION
**Pull request by ahmadelmowafy**
prep-javascript-objects solution

### CODE REVIEW

File: prep-javascript-objects/main.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  hobby: 'coding',
+};
+
+const fullName = person.firstName + ' ' + person.lastName;
+console.log(fullName);
+
+person.job = 'Neurosurgeon';
+console.log("The person's current job is:", person.job);
+
+person.previousJob = 'Prime Minister';
```
Comment (ahmadelmowafy) [inline prep-javascript-objects/main.js:13]: I forgot to use bracket notation when adding this property to the `person` object. It should've been written as `person['previousJob'] = 'Prime Minister';`

File: prep-javascript-objects/main.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+console.log("The person's previous job is:", person.previousJob);
```
Comment (ahmadelmowafy) [inline prep-javascript-objects/main.js:14]: Should've written it as `console.log("The person's previous job is:", person['previousJob']);`

File: prep-javascript-objects/prep-javascript-objects-quiz-notes.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+# prep-javascript-objects-quiz-notes
+
+## Quiz Questions
+
+Answer the following questions in the provided markdown file before turning in this exercise:
+
+- What data types can object properties hold?
```
Comment (ahmadelmowafy) [inline prep-javascript-objects/prep-javascript-objects-quiz-notes.md:7]: Strings, numbers, booleans, null, undefined, symbol, bigint

File: prep-javascript-objects/prep-javascript-objects-quiz-notes.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+- Describe the syntax (structure) of object-literals in JavaScript.
```
Comment (ahmadelmowafy) [inline prep-javascript-objects/prep-javascript-objects-quiz-notes.md:9]: Use curly braces (`{}`) to surround the keys and values in the object. Within the braces, add each key followed by a colon (`:`), followed by its value. This creates a key/value pair. Each key/value pair is separated by a comma (`,`).

**Review by robgmerrill — APPROVED** [review]
👍

**Pull request merged**